packages/aws-cdk/lib/api/deployments/asset-publishing.ts

```diff
@@ -241,10 +183,10 @@ export const EVENT_TO_LOGGER: Record<EventType, (x: string) => void> = {
 };
 
 class PublishingProgressListener implements IPublishProgressListener {
-  constructor(private readonly quiet: boolean) {}
+  constructor() {}
```

Comment:
Removed unused `quiet` option upstream in favor of it always being the default (`false`).